src/wp-includes/blocks.php

```diff
@@ -787,7 +786,14 @@ function make_before_block_visitor( $context ) {
 			// Candidate for first-child insertion.
 			$relative_position  = 'first_child';
 			$anchor_block_type  = $parent_block['blockName'];
-			$hooked_block_types = array_keys( get_hooked_blocks( $anchor_block_type, $relative_position ) );
+			$hooked_block_types = get_hooked_blocks( $anchor_block_type );
+
+			if ( isset( $hooked_block_types[ $relative_position ] ) ) {
+				$hooked_block_types = $hooked_block_types[ $relative_position ];
+			} else {
+				$hooked_block_types = array();
+			}
```

Comment:
```suggestion
			$hooked_block_types = isset( $hooked_block_types[ $relative_position ] ) ? $hooked_block_types[ $relative_position ] : array();
```

Could we use ternary operator here?

🔢 Same applies to other tests also

Reply:
Sure! It seemed too long when I was still using `array_key_exists`, but with `isset`, it looks a lot nicer 👍